resources/views/community/components/forum/post-comment-meta.blade.php

```diff
@@ -40,6 +52,23 @@
 @endif
 
 <p class='smalltext !leading-[14px]'>
-    {{-- Keep this all on a single line so white space isn't added before the comma --}}
-    {{ $formattedPostTimestamp }}@if($formattedEditTimestamp)<span class='italic smalltext !leading-[14px]'>, <span class='hidden sm:inline'>last</span> edited {{ $formattedEditTimestamp }}</span> @endif
+    <span 
+        title="{{ $shouldUsePostedTimeAgoDate ? $postCreatedTimestamp : null }}" 
+        class="{{ $shouldUsePostedTimeAgoDate ? "cursor-help" : null }}"
```

Comment:
This logic ends up with blank title and class attributes. 
```
<span
    title=""
    class=""
>
    
  July 12 2023, 5:22pm    </span>
```

Shouldn't they just be excluded entirely? Will specifying an empty class eliminate any default classes defined in the css?

Reply:
This was my React.js brain trying to do something with Laravel Blade. In React, passing `null` to an attribute causes that attribute to not be sent to the DOM. But this doesn't seem to be the case for Blade!

After studying for a little bit, I've learned the following syntax in Blade is valid and achieves what we're looking for:

```
<span
    @if($shouldUseEditedTimeAgoDate)
        class="cursor-help"
        title="{{ getNiceDate(strtotime($postEditedTimestamp)) }}"
    @endif
>
```